kratos/utilities/variable_utils.cpp

```diff
@@ -530,6 +530,81 @@ KRATOS_API(KRATOS_CORE) void VariableUtils::SetSolutionStepValuesVector(
         );
 }
 
+
+
+
+
+
+
+KRATOS_API(KRATOS_CORE) Vector VariableUtils::GetValuesVector(
```

Comment:
Why KRATOS_API, with defining in the class declaration is enought

Reply:
I think it is not required if the class have it, right @roigcarlo ?

Reply:
well.. i would prefer not to change it in this specific PR.

however...could it change if the methods are static?

Reply:
Static in a namespace yes, it is the only way. Static in a class already exported is not required

Reply:
well, i am not telling you are wrong... but i would keep this as it is, If you like we could try to take out all the KRATOS_API stuff in a different PR